Makefile

```diff
@@ -39,8 +39,9 @@ ifeq ($(NOBUSTER), 0)
 endif
 ifeq ($(NOBULLSEYE), 0)
 	BLDENV=bullseye make -f Makefile.work $@
+else
+	make -f Makefile.work $@
 endif
-	BLDENV=bullseye make -f Makefile.work docker-cleanup
```

Comment:
This part has been merged into master branch.

Reply:
It is added as a default rule in the Makefile.work for each distro build(stretch, buster and bullseye)